### PR TITLE
basic wiiu gamepad support in wii vc using libwiidrc

### PR DESCRIPTION
**Pull request by FIX94**
hope I didnt make any major mistakes with it.

### CODE REVIEW

**Comment by Mystro256** [comment]
Thanks! Ill take a look into the code right now

File: Gamecube/gc_input/controller-WUPC.c [hunk 1]
```diff
@@ -58,8 +58,8 @@ static button_t buttons[] = {
 	{ 13, WPAD_CLASSIC_BUTTON_PLUS, "+" },
 	{ 14, WPAD_CLASSIC_BUTTON_MINUS, "-" },
 	{ 15, WPAD_CLASSIC_BUTTON_HOME, "Home" },
-	{ 16, WUPC_EXTRA_BUTTON_RSTICK, "RS-Button" },
-	{ 17, WUPC_EXTRA_BUTTON_LSTICK, "LS-Button" },
+	{ 16, WUPC_EXTRA_BUTTON_RSTICK, "LS-Button" },
```
Comment (Mystro256) [inline Gamecube/gc_input/controller-WUPC.c]: Were you trying to reorder the values here? If so, I would assume WUPC_EXTRA_BUTTON_RSTICK should not map to LS-Button and vice versa

File: Gamecube/libgui/Gui.cpp [hunk 2]
```diff
@@ -128,7 +128,11 @@ void Gui::draw()
 					*(volatile unsigned int*)0xCC003024 = 0;  //reboot
 			  }
 #else
-				rld();
+				if(*(volatile unsigned int*)0x80001804 == 0x53545542 &&
```
Comment (Mystro256) [inline Gamecube/libgui/Gui.cpp]: I'm all for code that works, but can you add a comment here about these "magic numbers", or add some defines? I'm not familiar with what this is memory space that's dereferenced and the value it's expecting it to be. I'd rather not make this code more messy than it already is.

**Comment by FIX94** [comment]
good catch on that, corrected that now, also added a comment to that part and adjusted the gamepad stick to better values.

**Comment by Mystro256** [comment]
Thanks! Looks good to me, I'll do some testing and work on getting a release/build out.

**Comment by Mystro256** [comment]
Good catch, I was about to pull it. Ill take one more looksee.

**Comment by Mystro256** [comment]
I have no means of testing this right now, but please feel free to let me know if you see any bugs

**Pull request merged**